website/documentation/appendix/releasenotes/0.10.0-M1.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: documentation
+title: Release Notes
+---
+{% include base.html %}
+## Release Notes - 0.10.0-M1
+
+We are proud to announce the release of 0.10.0-M1.  
+Here are some of the major highlights:
+
+- **Web Editor** that allows to create information, function blocks and entity type models through the web browser
+- New **Code Generators** have been added to the list of existing generators.
+- Bug fixes and improvements
+
+
+### Web Editor
+With the **Web Editor**, you can create individual projects where you can describe the characteristics and functionality of the Information Models and publish then to Vorto Repository for others who could use to generate code using Code Generators. This is solely done using a web browser.
+
+  ![WebEditor]({{base}}/img/documentation/webeditor_details.png){:width="700px"}
+  
+### Code Generators
+
+We have three new Code Generators out of the box, those are
+
+* Bosch IoT Suite
+* Eclipse Kura
+* Web-based Device Dashboard
+
+#### Bosch IOT Suite
+
+This **Bosch IOT Suite** generator allows you to choose and  generate client code for **Device Simulator**, **Alexa Skillset** and **Device Validation** types.
+
+  ![Generators]({{base}}/img/documentation/bosch-iot_generator_configuration.png){:width="700px"}
+  
+#### Eclipse Kura
+
+This **Eclipse Kura** generator allows you to choose and generate client code for **Drivers (eg. Bluetooth)** and **Cloud Platform Connector for Bosch IoT Suite** types.
```

Comment:
Eclipse Kura generator supports the generation of a Kura gateway OSGI bundle that reads device data via bluetooth LE technology and sends the data to either Kura built-in Cloud Services or the Bosch IoT Suite.